.github/workflows/r.yml

```diff
@@ -53,7 +53,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        r: ["3.6"]
+        r: ["4.0"]
         ubuntu: [18.04]
```

Comment:
I don't think this was listed on the ticket, but as good a time as any to bump this from 18.04 to 20.04.